modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingLocalServiceImpl.java

```diff
@@ -1669,6 +1670,56 @@ protected void sendNotification(
 		}
 	}
 
+	protected CalendarBooking splitCalendarBooking(
+			CalendarBooking calendarBooking, java.util.Calendar splitJCalendar)
+		throws PortalException {
+
+		long laterStartTime = JCalendarUtil.convertTimeToNewDay(
+			calendarBooking.getStartTime(),
+			splitJCalendar.getTimeInMillis() + Time.DAY);
+
+		long duration =
+			calendarBooking.getEndTime() - calendarBooking.getStartTime();
+
+		long laterEndTime = laterStartTime + duration;
+
+		CalendarBooking laterCalendarBooking = addCalendarBooking(
+			calendarBooking.getUserId(), calendarBooking.getCalendarId(),
+			getChildCalendarIds(
+				calendarBooking.getCalendarBookingId(),
+				calendarBooking.getCalendarId()),
+			CalendarBookingConstants.PARENT_CALENDAR_BOOKING_ID_DEFAULT,
+			calendarBooking.getRecurringCalendarBookingId(),
+			calendarBooking.getTitleMap(), calendarBooking.getDescriptionMap(),
+			calendarBooking.getLocation(), laterStartTime, laterEndTime,
+			calendarBooking.getAllDay(), calendarBooking.getRecurrence(),
+			calendarBooking.getFirstReminder(),
+			calendarBooking.getFirstReminderType(),
+			calendarBooking.getSecondReminder(),
+			calendarBooking.getSecondReminderType(),
+			ServiceContextThreadLocal.getServiceContext());
+
+		deleteCalendarBookingInstance(
+			calendarBooking, splitJCalendar.getTimeInMillis(), true, false);
+
+		Recurrence laterRecurrenceObj = laterCalendarBooking.getRecurrenceObj();
+
+		for (java.util.Calendar exceptionJCalendar :
+				new ArrayList<>(laterRecurrenceObj.getExceptionJCalendars())) {
+
+			if (!JCalendarUtil.isLaterDay(exceptionJCalendar, splitJCalendar)) {
+				laterRecurrenceObj.removeExceptionJCalendar(exceptionJCalendar);
+			}
+		}
```

Comment:
@mbowerman Did you find a bug that required removing the exceptions predating the later date? I know they make no sense (since the event had not even started yet) but I wonder if these exceptions actually caused an undesired behavior. (It is mostly to understand the code, I believe these lines cause no problem.)

cc @natocesarrego